src/main/java/net/dries007/tfc/world/biome/TFCBiomes.java

```diff
@@ -7,6 +7,8 @@
 package net.dries007.tfc.world.biome;
 
 import java.util.Objects;
+
+import net.dries007.tfc.world.surface.builder.*;
```

Comment:
Need to fix this in your IDE or code style settings. We prefer not to use wildcard imports.